sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -275,6 +280,21 @@ public boolean contains(String name) {
         return configurations.containsKey(name);
     }
 
+    /**
+     * Determines if multiple configurations all exist.
+     *
+     * @param names Names of the configuration.
+     * @return True if configurations all exist, otherwise false.
+     */
+    public boolean containsAll(String... names){
```

Comment:
I'm not explicitly against this change but I do question how often it will be used outside of the context of `EnvironmentCredential`. Any chance this could be made into a local private method in the interim and if it is needed in additional locations in the future it gets added here.

Reply:
Removed from `Configuratoin`, now a private method in `EnvironmentCredential`.